package.json

```diff
@@ -5,18 +5,38 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "eslint-check": "eslint --ignore-path .gitignore . --fix"
+    "lint": "eslint --ignore-path .gitignore . --fix",
+    "build": "webpack --mode production",
+    "serve": "node server.js",
+    "start": "npm run build && npm run serve",
+    "dev": "webpack-dev-server --mode development & nodemon server.js "
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "express": "^4.17.1",
     "jquery": "^3.5.1",
     "moment": "^2.24.0",
-    "ol": "^5.3.3"
+    "ol": "^6.3.1"
   },
   "devDependencies": {
     "eslint": "^6.8.0",
-    "eslint-config-google": "^0.14.0"
+    "eslint-config-google": "^0.14.0",
+    "webpack-dev-server": "^3.1.3",
+    "@babel/core": "^7.0.0",
```

Comment:
Check to see if babel is necessary/if it requires more configuration to do anything.